bot.py

```diff
@@ -56,19 +57,215 @@
     embed.clear_fields()
 
 
+async def displayEmbedCountries(message, playerNumber, countries):
+    global hoi_dict
+    embed = discord.Embed(
+        title='Player ' + str(playerNumber + 1),
+        colour=random.randint(0, 0xffffff)
+    )
+    allImages = []
+    allCountries = []
+    for cou in hoi_dict:
```

Comment:
typically variable names like `cou` are poor choices.  It is hard for a new reviewer to understand what the variable is meant to hold. I recommend more meaningful names.